docs/integrations/apps/guide/best-practices.mdx

```diff
@@ -37,20 +37,23 @@ Rather than polling endpoints, get notified when updates occur by subscribing to
 
 ### Manage user session timeouts
 
-Add BigCommerce's JavaScript SDK to your single-click app's front-end to prevent users from getting logged out of the control panel while using your app. To do so, reference the following script in your app's client-side code:
-
-```js copy
-      https://cdn.bigcommerce.com/jssdk/bc-sdk.js
-```
-
-To perform some action when a logout occurs, specify an `onLogout` callback:
+The [BigCommerce SDK](https://www.npmjs.com/package/@bigcommerce/app-sdk-js) keeps apps in sync with the control panel. To consume the SDK, you should load it at the top of an application as a part of the main index.
 
+Run the following command to import the library:
 
 ```js copy
-Bigcommerce.init({
-      onLogout: callback
-});
+import 'bigcommerce-app-sdk-js';
+```
+Next, initialize the SDK. This should be done in the first part of the application that has access to the window object. In React-based applications, including NextJS, this can be done inside of `useEffect` or `componentDidMount` (for class components).
+
+``` js copy 
+window.bcAsyncInit = function() {
+  Bigcommerce.init({
+    onLogout: callback
+  });
+}
 ```
+Finally, to add an additional layer of security, you can pass an onLogout callback. This callback will be hit if the user logs out of the control panel, including logging out from a separate tab. For example, the NextJS sample app calls an internal API which removes the user from the active session.
```

Comment:
We could link an example
https://github.com/bigcommerce/sample-app-nodejs/blob/6f779f075320059c97452e10f91c4e51d2829324/scripts/bcSdk.js#L19